src/imitation/rewards/discrim_net.py

```diff
@@ -196,13 +196,12 @@ def reward_train(
         returns `log_act_prob`, the generator's log action probabilities.
         `log_act_prob[i]` is equal to the generator's log probability of
         choosing `act[i]` given `obs[i]`.
-        `np.squeeze(log_act_prob)` has shape `(batch_size,)`.
     Returns:
         The rewards. Its shape is `(batch_size,)`.
     """
     del steps
-    log_act_prob = np.squeeze(
-      gen_log_prob_fn(observation=obs, actions=act, logp=True))
+    log_act_prob = np.reshape(
```

Comment:
Does `np.flatten` do what we want here? That seems more canonical than `np.reshape(..., (-1,))`.

Reply:
Oh, that's right. Replaced `reshape` with `flatten`.